assignment.js

```diff
@@ -66,6 +74,20 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+
+
+const numbers = [57, 13, 4, 38]
+let currentLatgestNumber = 0
+
+for (i = 0; i < numbers.length; i++) {
+    if (numbers[i] > currentLatgestNumber) {
+        currentLatgestNumber = numbers[i]
+    }
+}
+console.log(currentLatgestNumber)
+
+
 // Add your code above this line
 
```

Comment:
@jl1982 Good job here utilizing a for loop to loop through each element, check for the largest number and store the value in a variable. One suggestion for your loops conditionals and functions the industry standard is to indent two spaces, similar to how you indented in your while loop.
check out another approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)